src/game/music.c

```diff
@@ -62,20 +62,16 @@ bool Music_Play(int16_t track)
         return false;
     }
 
-    if (track <= 1) {
-        return false;
-    }
-
     if (g_Config.fix_secrets_killing_music && track == MX_SECRET) {
         return Sound_Effect(SFX_SECRET, NULL, SPM_ALWAYS);
     }
 
-    if (track == 0) {
+    if (track <= 1) {
```

Comment:
Can we change this to `MX_UNUSED_1` as well?

Also I think rr's original point about the possibility of a music trigger with track ID 0 or 1 existing in a level is still valid. In this new code, a trigger for 0 or 1 will now stop any music being played rather than just ignoring it. If we remove this block (since it was always unused) and just use the
```
if (track <= 1) {
        return false;
    }
```
above, I think that would be a better solution?

Reply:
@carlo-bramini what do you think of my suggestion?